src/Solvers/Random.jl

```diff
@@ -1,8 +1,64 @@
+using OptimizedEinsum: removedsize, removedsize_noise, ssa_path_cost, ContractionPath
+using OptimizedEinsum.Solvers: greedy_choose_simple!, greedy_choose_thermal!
+using Base: @kwdef
+using Random: seed!
+
 abstract type RandomSolver <: Solver end
 
-struct RandomGreedy <: RandomSolver
-    temperature::Float32
-    rel_temperature::Bool
-    nbranch::Int
-    cost_fn::Core.Function
+@kwdef struct RandomGreedy <: RandomSolver
+    temperature::Float32 = 1.
+    rel_temperature::Bool = true
+    nbranch::Int = 8
+    max_time::Number = 100
+    repeats::Int = 128
+    choose_fn::Function = (x, y) -> greedy_choose_thermal!(x, y, nbranch, temperature, rel_temperature)
+    cost_fn::Function = removedsize_noise
+    minimize::String = "flops" # "flops" or "size"
```

Comment:
On these cases, `Symbol` is usually used.

```suggestion
    minimize::Symbol = :flops # "flops" or "size"
```